git-hg/jdk8u/merge.sh

```diff
@@ -108,6 +109,44 @@ function updateRepo() {
 
 }
 
+# Builds a local repo on a new machine by pulling down the existing remote repo
+function rebuildLocalRepo() {
+    hgRepo=$1
+
+    # Steps required to build a new host
+    #
+    # 1. Clone upstream mirrors (done by updateMirrors function)
+    #
+    # 2. Pull adopt repo down into $REPO
+    #
+    # 3. Set up remotes on $REPO
+    #     Remotes should look as follows:
+    #       upstream: git@github.com:AdoptOpenJDK/openjdk-jdk8u.git (or aarch)
+    #       root:    "$MIRROR/root/"
+    #       origin:  "$MIRROR/root/"
+    #
+
+    # Step 1 Clone mirrors
+    updateMirrors $hgRepo
```

Comment:
Does that clone or update or is that function a clone and/or update?

Reply:
yes if it does not exist it creates it, and updates it